packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -508,4 +508,3 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
     this.hasEndActions = slotChangeHasAssignedElement(event);
   };
 }
-
```

Comment:
can you undo these unrelated formatting changes? to files in `packages/calcite-components`

Reply:
These were auto fixes. I am able to fix the props-sorting in action-bar but the spacing changes reassert themselves every time I save.

Reply:
I can not. These changes are coming from the husky lint rules on commit which auto-fix basic linting errors.

Reply:
You can do `git commit -m "cleanup" --no-verify` to skip git's precommit hook

Reply:
Note this is important so the PR doesn't show up in CC's changelog as a feature

Reply:
I reverted the formatting changes in a separate branch. Can you cherry pick [this commit](https://github.com/Esri/calcite-design-system/commit/b7ae4409c4986619b49bd48097f178e58f5109ee) (b7ae440) ?

Reply:
Or we can wait until #7774 is installed and then you can update this branch. Because using `--no-verify` is bad practice.